spec/integration/bisect_spec.rb

```diff
@@ -32,5 +32,13 @@ def bisect(cli_args, expected_status=nil)
         expect(output).to include("Bisect failed!", "The example ordering is inconsistent")
       end
     end
+
+    context "when the bisect command is long" do
+      it 'does not hit pipe size limit and does not get stuck' do
+        output = bisect(%W[spec/rspec/core/resources/blocking_pipe_bisect_spec.rb_], 1)
+        puts output
```

Comment:
We don't need the `puts output` committed 🙃

Reply:
Olalala. Sorry.